src/Services/Shop/Shopify.php

```diff
@@ -208,6 +209,41 @@ public function customer($id): Customer
         return $this->mapCustomer($customer);
     }
 
+    /**
+     * @param array $data
+     * @return array
+     */
+    public function storeDiscount(array $data): array
+    {
+        $required_fields = ['title', 'value_type', 'value', 'starts_at'];
+
+        foreach ($required_fields as $field) {
```

Comment:
Je kan hier tenminste 1 testcase voor schrijven:
itCantStoreDiscountMissingTitle
- je roept hier de storeDiscount method aan
- je catched de error 
- je checked of de error message uit de excecption overeen komt met 'price_rule.title is a required field'